Dockerfile copy

```diff
@@ -0,0 +1,38 @@
+FROM quay.io/ibmgaragecloud/gradle:jdk11 AS builder
```

Comment:
remove this file?

Reply:
yes I noticed this is no longer used, I'll delete it, Thanks.